### PR TITLE
Update azdo_windows.yml

### PR DESCRIPTION
**Pull request by brodiep21**
updated sdk version to latest reflection app sample version, updated paths

### CODE REVIEW

**Comment by brodiep21** [comment]
> @dotnet-policy-service agree

File: 8.0/Apps/WeatherTwentyOne/devops/AzureDevOps/azdo_windows.yml [hunk 1]
```diff
@@ -47,16 +47,16 @@ stages:
...
 
     # Signing Android App : https://docs.microsoft.com/en-us/azure/devops/pipelines/apps/mobile/app-signing?view=azure-devops&tabs=apple-install-during-build#sign-your-android-app
 
     - task: CopyFiles@2
       inputs:
         Contents: |
-          **\**\6.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-android\publish\*.apk
-          6.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-android\publish\*.apk
+          **\**\8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-android\publish\*.apk
+          8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-android\publish\*.apk
```
Comment (jfversluis) [inline 8.0/Apps/WeatherTwentyOne/devops/AzureDevOps/azdo_windows.yml:59]:
```suggestion
          **\**\8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net8.0-android\publish\*.apk
          8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net8.0-android\publish\*.apk
```

File: 8.0/Apps/WeatherTwentyOne/devops/AzureDevOps/azdo_windows.yml [hunk 2]
```diff
@@ -107,18 +107,18 @@ stages:
     - task: CopyFiles@2
       inputs:
         Contents: |
-          **\**\6.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-windows10.0.19041.0\win10-x64/**/*.msix
-          6.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-windows10.0.19041.0\win10-x64/**/*.msix
+          **\**\8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-windows10.0.19041.0\win10-x64/**/*.msix
+          8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net7.0-windows10.0.19041.0\win10-x64/**/*.msix
```
Comment (jfversluis) [inline 8.0/Apps/WeatherTwentyOne/devops/AzureDevOps/azdo_windows.yml:111]:
```suggestion
          **\**\8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net8.0-windows10.0.19041.0\win10-x64/**/*.msix
          8.0\Apps\WeatherTwentyOne\src\WeatherTwentyOne\bin\Release\net8.0-windows10.0.19041.0\win10-x64/**/*.msix
```

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Thank you!

**Pull request merged**